scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -572,7 +569,7 @@ class FossId internal constructor(
      * Wait until the repository of a scan with [scanCode] has been downloaded.
      */
     private suspend fun waitDownloadComplete(scanCode: String) {
-        val result = wait(WAIT_INTERVAL_MS * WAIT_REPETITION, WAIT_INTERVAL_MS) {
+        val result = wait(WAIT_INTERVAL_MS * config.waitRepetition, WAIT_INTERVAL_MS) {
```

Comment:
Should the download use the same timeout? I would assume that this would be much faster than a scan operation.

Reply:
You're right. However don't you think keeping only one timeout lower the complexity of the code ?